src/test/java/uk/ac/ebi/atlas/search/GeneSearchServiceIT.java

```diff
@@ -52,12 +54,14 @@ void cleanDatabaseTables() {
         ResourceDatabasePopulator populator = new ResourceDatabasePopulator();
         populator.addScripts(
                 new ClassPathResource("fixtures/experiment-delete.sql"),
-                new ClassPathResource("fixtures/scxa_marker_genes-delete.sql"));
+				new ClassPathResource("fixtures/scxa_cell_group-delete.sql"),
+                new ClassPathResource("fixtures/scxa_cell_group_marker_genes-delete.sql"));
         populator.execute(dataSource);
     }
 
-    @ParameterizedTest
+    @Ignore
     @MethodSource("experimentAccesionWithoutPreferredKProvider")
+//	@ParameterizedTest
```

Comment:
Shouldn't it be `@ParameterizedTest`? Any reason it has been commented out?

Reply:
Yes, as per @alfonsomunozpomer, the Clustering algorithm has been changed and we don't have a file that contains all k values that are false boolean flag. He said that we will have a discussion with the curation team or bioinformatics team to find the right file location in the new year. Asked me to ignore.